monkey/monkey_island/cc/ui/src/components/pages/MapPage.js

```diff
@@ -138,16 +150,37 @@ class MapPageComponent extends AuthComponent {
     );
   }
 
+  handleScroll(e) {
+    let element = e.target;
+
+    let telemetryStyle = window.getComputedStyle(element)
+    let telemetryLineHeight = parseInt((telemetryStyle.lineHeight).replace('px', ''))
+
+    this.setState({
+      isScrolledUp: !!(element.scrollTop < this.scrollTop),
+      telemetryCurrentLine: parseInt(element.scrollTop/telemetryLineHeight)+1,
+      telemetryLines: parseInt(element.scrollHeight/telemetryLineHeight)
```

Comment:
Do we need the `parseInt` here?
How about 
```suggestion
      telemetryCurrentLine: Math.round(element.scrollTop/telemetryLineHeight)+1,
      telemetryLines: Math.round(element.scrollHeight/telemetryLineHeight)
```